cmake/faceliftMacros.cmake

```diff
@@ -211,6 +211,8 @@ function(facelift_generate_code )
             list(APPEND BASE_CODEGEN_COMMAND "--all")
         endif()
 
+        list(APPEND BASE_CODEGEN_COMMAND "--no_moc_path" "${ARGUMENT_OUTPUT_PATH}")
```

Comment:
no new parameter is needed. The file should be generated in the same folder as the generated C++ files

Reply:
this is needed for correct paths in HEADERS_NO_MOC_FROM_FILE